sensor_msgs/Image input_image

---
sensor_msgs/Image output_image
bool success